assets/prototype/domain/eventEditor/containers/queries/useEventId.js

```diff
@@ -0,0 +1,7 @@
+const useEventId = () => {
+	const eventData = window.eeEditorEventData;
+	const { eventId } = eventData;
+	return eventId;
+};
```

Comment:
as discussed in chat, this should pull from a Context Provider (or Apollo)
but that can be done as part of #1955 

see: https://github.com/eventespresso/event-espresso-core/issues/1955#issuecomment-562208734